infra/base-images/base-runner/test_all.py

```diff
@@ -80,6 +80,11 @@ def find_fuzz_targets(directory):
       continue
     if filename.startswith('jazzer_'):
       continue
+    if os.getenv('FUZZING_ENGINE') == 'centipede':
+      if os.getenv('SANITIZER') in filename and os.path.isdir(path):
```

Comment:
Yep, I can make it a bit tighter: The name of that directory should always be `${PROJECT_NAME}_${SANITIZER}`.
IIRC, the `${PROJECT_NAME}` is not recorded in any env variable in that docker container, but I should be able to get it from the *parent directory name* (which is a bit hacky).
Then I will make this condition look for that directory name specifically.

Reply:
Unfortunately, I cannot get `${PROJECT_NAME}` because that directory name was mapped to an arbitrary string in the docker container.
So I matched against `_${SANITIZER}` instead.